tools/contrib/junit_xml/__init__.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
```

Comment:
Can we have a thin shell script wrapper around test.py that sets up the virtualenv, then?  Seems preferable to having third party code in the repo.